src/inverseRobot.js

```diff
@@ -7,7 +7,19 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const inverse = {};
+
+  const keys = Object.keys(robot);
+
+  for (const key of keys) {
+    if (inverse.hasOwnProperty(robot[key])) {
+      return null;
+    } else {
+      inverse[robot[key]] = key;
+    }
```

Comment:
else statement is redudant
```suggestion
    if (inverse.hasOwnProperty(robot[key])) {
      return null;
    }
    
    inverse[robot[key]] = key;

```

Reply:
Yes. I agree. Thank you for comment!